src/import-map-injector.ts

```diff
@@ -77,6 +77,10 @@ injectorImportMaps.forEach((scriptEl) => {
 if (window["importMapOverrides"]) {
   importMapJsons.push(window["importMapOverrides"].getOverrideMap());
   importMapJsons.push(window["importMapOverrides"].getOverrideScopes());
+  if (window["importMapOverrides"].getExternalOverrideMap) {
+    // I think it should always have this, but just in case...
+    importMapJsons.push(window["importMapOverrides"].getExternalOverrideMap());
+  }
```

Comment:
We don't need to support versions of import-map-overrides that don't have the `getExternalOverrideMap` function

```suggestion
  importMapJsons.push(window["importMapOverrides"].getExternalOverrideMap());
```